src/jarabe/journal/listview.py

```diff
@@ -747,7 +747,11 @@ def __icon_clicked_cb(self, cell, path):
 
     def __cell_title_edited_cb(self, cell, path, new_text):
         iterator = self._model.get_iter(path)
-        self._model[iterator][ListModel.COLUMN_TITLE] = new_text
+        if new_text.isspace() or new_text == '':
+            return
+        else:
+            self._model[iterator][ListModel.COLUMN_TITLE] = new_text
+
         self.emit('title-edit-finished')
```

Comment:
Hum, I just noticed that if you try and rename it to something empty, this signal will not be emitted.  This is an issue, eg. try:

1.  Go into name editing mode in the list view
2.  Rename to "" (empty)
3.  Press enter to confirm
4.  Try typing to search in the journal (don't click in the search entry, that should not be needed), notice that doesn't work though - the entry is not autofocused on keypress